schemas/fig.d.ts

```diff
@@ -29,8 +29,14 @@ declare namespace Fig {
   // both set to void by default
   export type StringOrFunction<T = void, R = void> = string | Function<T, R>;
 
+  export type VersionedSpec = (version: string | undefined) => Spec;
```

Comment:
This can be simplified a bit,  maybe overkill:
```suggestion
  export type VersionedSpec = (version?: string) => Spec;
```